runtimes/adapters/goose.py

```
@@ -132,10 +134,32 @@ async def execute(self, spec: TaskSpec) -> TaskResult:
+        model = spec.model_preference or self._model
+        api_base: str | None = None
+        _needs_browser = task_wants_browser(spec)
+        if _needs_browser:
+            try:
+                from providers.kimi_bridge import kimi_bridge_runtime_config
+                _kb = kimi_bridge_runtime_config()
+                if _kb:
+                    model = str(_kb.get("model", model))
+                    api_base = str(_kb.get("base_url", "")) or None
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)